_includes/supporter.html

```diff
@@ -2,14 +2,15 @@
   <div class="page-details">
     <div class="title">Sponsors</div>
     <div class="description">
-      PyCon India is entirely driven by volunteers.
-      Sponsoring the event helps to sustain and grow
-      the conference as well as the community.
-      Sponsors help in making the conference affordable,
-      and maintaining the inventory for the conference.
+      PyCon India is a volunteer-driven event. 
+      Sponsoring the event helps bring the volunteers effort to fruition, 
+      sustain, and improve the conference and grow the community. 
+      Sponsors help us make the conference affordable, 
+      bring more facilities to our attendees, 
+      and maintain the inventory for the conference.
     </div>
     <p class="description">
-      Are you interested to support PyCon India 2020? Reach us out at 
+      Interested in supporting PyCon India 2020? Reach out to us at 
```

Comment:
Isn't the first question an incomplete sentence?

Reply:
Ah yes. 
How about `Reach out to us at <link> to support PyCon India 2020`?

Reply:
ping @palnabarun